R/deprecated.R

```diff
@@ -0,0 +1,57 @@
+#' DEPRECATED: Get the comments (as a list)
+#'
+#' @param urlname The name of the group.
+#' @param api_key Your api key.
+#' @param event_id The id of the event.
+#'
+#' @examples
+#' \dontrun{
+#' urlname <- "rladies-san-francisco"
+#' api_key <- Sys.getenv("MEETUP_KEY")
+#' past_events <- get_events(urlname = urlname,
+#'                       api_key = api_key,
+#'                       event_status = "past")
+#' event_id <- past_events[[1]]$id
+#' comments <- get_meetup_comments(urlname, api_key, event_id)
+#'}
+#' @export
+get_meetup_comments <- function(urlname, api_key, event_id) {
```

Comment:
AH I understand now the `.Deprecated()` function -- sorry I wasn't able to incorporate it when you suggested it before 🙈